editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { Component } from 'element';
+import PanelBody from 'components/panel-body';
+import PanelBodyToggle from 'components/panel-body-toggle';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+
+class PostStatus extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			opened: true,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
+		this.setState( {
+			opened: ! this.state.opened,
+		} );
+	}
+
+	render() {
+		return (
+			<PanelBody>
+				<PanelBodyToggle
+					onClick={ this.toggle }
+					opened={ this.state.opened }
+					label={ __( 'Status & Visibility' ) }
+				/>
+				{ this.state.opened &&
+					<div>
+						<div className="editor-sidebar-post-status__row">
+							<span>{ __( 'Pending review' ) }</span>
+							<FormToggle
+								checked={ false }
+								onChange={ () => {} }
```

Comment:
>I'm on the fence. We're not using it anywhere and I'm not sure about the future of this feature in React.

Which feature? `defaultProps`? If it were to be deprecated, there'd still be some mechanism for defining defaults, likely through destructuring:

```js
const { onChange = noop } = this.props;
```

I note here because it seems odd to me we'd _prefer_ to have an error thrown when the handler isn't defined.